core/domain/suggestion_registry_test.py

```diff
@@ -956,6 +947,14 @@ def test_pre_update_validate_change_question_dict(self):
             'skill_id': 'skill_1'
```

Comment:
Make it more obvious that this change is different, for example:

```python
different_change = copy.copy(expected_suggestion_dict['change'])
del different_change['linked_skill_ids']

suggestion = (...)

# Adding questions will fail because the change is different.
with self.assertRaisesRegexp(...):
  # ...
```

Reply:
This test will fail because the change is same not different. Not sure what you mean though

Reply:
I manually diffed `self.suggestion_dict` and the local `change` here, and your local one is missing `linked_skill_ids`. Are you saying that this was unintentional?

Reply:
Done as we discussed offline